<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose your future. Choose life.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="propLayout">
       <item>
        <widget class="QTableView" name="propView">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="propEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="propBtnLayout">
         <item>
          <widget class="QPushButton" name="addOrdButton">
           <property name="text">
            <string>Add property</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addCatButton">
           <property name="text">
            <string>Add checklist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addBigButton">
           <property name="text">
            <string>Add text</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tagWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tagView">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tagEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="tagBtnLayout">
         <item>
          <widget class="QPushButton" name="addTagButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeTagButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
